doc/release/1.4.0-notes.rst

```diff
@@ -51,13 +90,18 @@ The function ``scipy.linalg.subspace_angles(A, B)`` now gives correct
 results for complex-valued matrices. Before this, the function only returned
 correct values for real-valued matrices.
 
+New boolean keyword argument ``check_finite`` for `scipy.linalg.norm`; whether to check that the input matrix contains only finite numbers. Disabling may give a performance gain, but may result in problems (crashes, non-termination) if the inputs do contain infinities or NaNs.
+
 
 `scipy.ndimage` improvements
 ----------------------------
 
 
 `scipy.optimize` improvements
 -----------------------------
+It is now possible to use linear and non-linear constraints with `scipy.optimize.differential_evolution`, this uses an approach by Lampinen (A constraint handling approach for the differential evolution algorithm. Proceedings of the 2002 Congress on Evolutionary Computation. CEC'02 (Cat. No. 02TH8600). Vol. 2. IEEE, 2002).
```

Comment:
I don't think the reference to a paper should be here, suggest to remove "this uses an approach ..."

Reply:
Removed, thanks